.travis.yml

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
@isht3 There are many things missing in this. Can you please take a look at EvalAI's Travis setup for reference.

Reply:
Yes, I did. I think many of that was for Postgres, Cache, Services, and Coveralls. Which I believe are not required now. Should I add the notification for slack?

Reply:
@isht3 I think we should also add this: https://docs.travis-ci.com/user/reference/trusty/#Container-based-with-sudo%3A-false

Reply:
Sorry, I didn't see this comment. I added it